crates/cast/bin/cmd/call.rs

```diff
@@ -254,11 +254,11 @@ mod tests {
     #[test]
     fn can_parse_call_data() {
         let data = hex::encode("hello");
-        let args: CallArgs =
-            CallArgs::parse_from(["foundry-cli", "--data", format!("0x{data}").as_str()]);
-        assert_eq!(args.data, Some(data.clone()));
+        let args = CallArgs::parse_from(["foundry-cli", "--data", data.as_str()]);
+        assert_eq!(args.data, Some(data));
```

Comment:
We don't strip prefixes by default anymore

Reply:
I think we should?

Reply:
ah will be taken care of by hex decoding anyway so is fine